### DISCUSSION TITLE
Limiting Exposure to Mass Scanners

### DISCUSSION THREAD

**Discussion by CCU7LSP** [telekom-security/tpotce#1494] [Q&A] [answered]
So, I went into the tpot.yml file and selected the honeypots I wanted to have active and which ports to be open to connection.

However, the mass scanners out there such as nmap and online port scanners, list like 50+ ports being open on my machine, when I thought I only had 12?

Has anyone found a work around for this and has an example on best practice to mitigate this unnecessary exposure to seem more legitimate?

**Comment by t3chn0m4g3** [comment]
You can activate the [Blackhole](https://github.com/telekom-security/tpotce/tree/alpha?tab=readme-ov-file#blackhole) feature.

**Reply by CCU7LSP** [reply]
Did I miss understand the feature of Blackhole then? I read its description as it will block all known mass scanners kinda like a blacklist. I'm not trying to outright ban them, just limit them to seeing the honeypot ports and not all these ports open for the docker services that get spun up? Someone already suggested utilizing iptables and specifying specific network rules.

**Reply by t3chn0m4g3** [reply, marked as answer]
Then you need to disable `honeytrap`.

**Reply by CCU7LSP** [reply]
Thank you sir, I didn't think to try Honeytrap as it didn't have a specific port labeled in the tpot.yml.

**Reply by t3chn0m4g3** [reply]
Thanks for the feedback, great to hear it works 😃
